src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,38 @@
+.Navbar {
+  width: 100%;
```

Comment:
the width should be either 90% (vw) and then margin: 0 auto;
or there should be some paddings, because there is some space at the both sides